RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+## Individual updates
+
+PR [#572](https://github.com/IAMconsortium/pyam/pull/572) unpinned the requirements for xlrd and added openpyxl as a requirement to ensure ongoing support of both `.xlsx` and `.xls` files out of the box
```

Comment:
Just before clicking merge, I had a another look at the release notes and realized I forgot to add the "Next Release" header after doing the release 1.1... Also added an explicit "Dependency changes" section for good measure...

```suggestion
# Next Release

## Dependency changes

The dependencies were updated to require `xlrd>=2.0` (previously `<2.0`) and `openpyxl` was added as a dependency.

## Individual updates

- [#572](https://github.com/IAMconsortium/pyam/pull/572) Unpinned the requirements for xlrd and added openpyxl as a requirement to ensure ongoing support of both `.xlsx` and `.xls` files out of the box
```